src/main.ts

```diff
@@ -31,16 +32,50 @@ function onShutdown() {
   process.exit(0);
 }
 
-async function main(cfg: IAppConfig) {
-  const server = new Server(app(cfg), {
+async function main() {
+  const cloudFoundryAPI = expectEnvVariable('API_URL');
+  let authorizationURL = process.env.AUTHORIZATION_URL;
+  let uaaAPI = process.env.UAA_URL;
```

Comment:
Can you describe why these would be overridden by environment variables?

It'd be simpler (easier to understand the code and any possible misconfigurations) if they weren't.